lab_1_keywords_tfidf/main.py

```diff
@@ -99,12 +138,21 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
-
-def calculate_expected_frequency(
-    doc_freqs: dict[str, int], corpus_freqs: dict[str, int]
-) -> Optional[dict[str, float]]:
+    if not(isinstance(term_freq, dict) and term_freq and isinstance(idf, dict)
+           and all(isinstance(k, str) for k in idf.keys()) and all(isinstance(v, float) for v in idf.values())):
+        return None
+    for key, value in term_freq.items():
+        if not (isinstance(key, str) and isinstance(value, float)):
+            return None
+        tfidf_d = {}
+        for word in term_freq:
+            if word not in idf.keys():
+                idf[word] = math.log(47 / 1)
```

Comment:
why `/ 1` ??

Reply:
And I want to ask about "Revert every docstring", should I add back strings with pass or what strings? I think other docstrings are at their place or not?

Reply:
ask about revert in comments about docs

Reply:
Why don't you do 0+1?